### PR TITLE
Modified Target user profile and Value proposition

### PR DESCRIPTION
**Pull request by wynonaK**
Modified Target user profile and Value proposition.
Removed 1 glossary term and added 1 glossary term.

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -782,13 +782,16 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing over mouse input
-* is reasonably comfortable using CLI apps
+* tech-savvy individual
```
Comment (Aquarinte) [inline docs/DeveloperGuide.adoc]: Perhaps it will be better to state "tech-savvy veterinary practioners"

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -782,13 +782,16 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 
 *Target user profile*:
```
Comment (Aquarinte) [inline docs/DeveloperGuide.adoc]: Or you can also consider stating "veterinarians and veterinary technicans" beside "target user profile" to be clear and upfront about our target user profile.

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -782,13 +782,16 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
...
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing over mouse input
-* is reasonably comfortable using CLI apps
+* tech-savvy individual
+* can type relatively fast while maintaining accuracy
+* prefers being on desktop rather than on mobile
+* generally comfortable with typing on CLI applications without clicking
+* is a veterinarian or assistant from a vet clinic in Singapore
```
Comment (Aquarinte) [inline docs/DeveloperGuide.adoc]: provide veterinary services to domestic animals in Singapore

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (chialejing) [reply docs/DeveloperGuide.adoc]: Agreed with @Aquarinte; I think "tech-savvy" veterinary practitioners" will be better, since it will make it very clear that we are targeting vets.

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -782,13 +782,16 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
...
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing over mouse input
-* is reasonably comfortable using CLI apps
+* tech-savvy individual
+* can type relatively fast while maintaining accuracy
+* prefers being on desktop rather than on mobile
+* generally comfortable with typing on CLI applications without clicking
```
Comment (chialejing) [inline docs/DeveloperGuide.adoc]: Maybe instead of writing "generally comfortable with typing...", it will be better to show that they need such speed and efficiency when dealing with large numbers of animals daily (and especially more so in the cases of emergency). So it's more of a necessity for them to be able to just type and search for records fast without even reaching for the mouse because they do not want to waste time finding buttons and clicking through the application. Maybe rephrase to something like "Requires a CLI application due to <reason>".

File: docs/DeveloperGuide.adoc [hunk 5]
```diff
@@ -782,13 +782,16 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
...
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing over mouse input
-* is reasonably comfortable using CLI apps
+* tech-savvy individual
+* can type relatively fast while maintaining accuracy
+* prefers being on desktop rather than on mobile
```
Comment (chialejing) [inline docs/DeveloperGuide.adoc]: For this, I am not very sure how applicable is the "prefers desktop over mobile" part - because usually in clinics, management of databases etc. is done on computers, no? (At least for private clinics / polyclinics that I have visited). So it's not a "preference", but rather a requirement. @Aquarinte maybe you can enlighten us on this? :)

Comment (Aquarinte) [reply docs/DeveloperGuide.adoc]: From what I have observed so far, both Vets and Vet technicians use desktops. They have to deal with a lot of information (e.g. appointments, medical histories, patient registrations etc), so desktop and physical keyboard are a more viable option compared to mobile.

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (wynonaK) [reply docs/DeveloperGuide.adoc]: ok, will change

File: docs/DeveloperGuide.adoc (hunk 5 above)
Comment (wynonaK) [reply docs/DeveloperGuide.adoc]: So I guess I will change this to "Uses a desktop for work"?

File: docs/DeveloperGuide.adoc [hunk 6]
```diff
@@ -782,11 +782,11 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 
 *Target user profile*:
 
-* tech-savvy individual
+* tech-savvy veterinary practioners
```
Comment (howardlondon) [inline docs/DeveloperGuide.adoc]: The term should be Veterinarian / Veterinarian Assistance

Comment (chialejing) [reply docs/DeveloperGuide.adoc]: *practitioners.

Howard's suggestion makes sense too. Because it is more specific and shows clearly who our target audience is.
